packages/editor/src/utils/media-upload/index.js

```diff
@@ -7,11 +7,7 @@ import { noop } from 'lodash';
  * WordPress dependencies
  */
 import { select } from '@wordpress/data';
-
-/**
- * Internal dependencies
- */
-import { mediaUpload } from './media-upload';
+import { utils } from '@wordpress/media-utils';
```

Comment:
Have you considered having:
```suggestion
import { mediaUpload } from '@wordpress/media-utils';
```

or 

```suggestion
import { utils } from '@wordpress/media';
```